dictionaries/en_US/src/en_US.txt

```diff
@@ -1960,6 +1960,7 @@ blasty
 blatter
 blattering
 blatters
+blazingly
```

Comment:
Please add new words to `src/additional_words.txt` instead of `en_US.txt`. `en_US.txt` is the original word list.

Reply:
Moved them to the right location